tests/test_layout_objects.py

```diff
@@ -440,16 +440,17 @@ def test_tab_and_tab_holder(self):
                     "one",
                     "first_name",
                     css_id="custom-name",
-                    css_class="first-tab-class active",
```

Comment:
I've added this fix because we don't have to add `active` css class manually. The tests should to check if the class added automatically.